manifests/nodejs.yml

```diff
@@ -7,8 +7,8 @@ tests/:
   appsec/:
     api_security/:
       test_api_security_rc.py:
-        Test_API_Security_RC_ASM_DD_processors: missing_feature
-        Test_API_Security_RC_ASM_DD_scanners: missing_feature
+        Test_API_Security_RC_ASM_DD_processors: v5.3.0
+        Test_API_Security_RC_ASM_DD_scanners: v5.3.0
         Test_API_Security_RC_ASM_processor_overrides_and_custom_scanner: missing_feature # waf does not support it yet
       test_apisec_sampling.py:
         Test_API_Security_sampling: missing_feature
```

Comment:
do we not havr that feature ?

Reply:
yes, and it is an XPASS. enabled, let's see the results

Reply:
Did you run the system-tests manually on the version you put in the manifest ? the CI doesn't check old versions

Reply:
I checked that the code was there, and now I run the test in my local, for all variants (including next)